chore: edge and light mode for modal dialog

deboer-tim opened the pull request:
### What does this PR do?

Adds light mode colors for Modal dialog along with a defined 1px border so that it is not washed out.

Initial light mode color values are taken from other mappings in the registry. It is not possible to use bg-opacity-60 with colors from the registry due to Tailwind CSS post processing order. I looked at several other options (defining colors as r g b instead of hex, trying several css functions, but the simplest is just to use direct styling instead.

### Screenshot / video of UI

Dark:

<img width="193" alt="Screenshot 2024-05-17 at 11 47 34 AM" src="https://github.com/containers/podman-desktop/assets/19958075/195c3cc1-acdb-412b-b5d5-99ad30171e3b">

Light:

<img width="193" alt="Screenshot 2024-05-17 at 11 46 51 AM" src="https://github.com/containers/podman-desktop/assets/19958075/0daa7753-9d55-4bf2-8bbb-c84e8c3f9746">

### What issues does this PR fix or reference?

Fixes #7187.
First minor part of #7184.

### How to test this PR?

Confirm no regressions and new border in dark mode, check colors (of Modal, not any contents) in light mode.

cdrage reviewed (changes requested):
![Screenshot 2024-05-22 at 9 47 23 AM](https://github.com/containers/podman-desktop/assets/6422176/9e4becaf-8d1f-428f-99fb-078947c687e2)

I'm finding it very difficult to read / as though the modal has some form of opacity? I saw the comment:  "It is not possible to use bg-opacity-60 with colors from the registry due to Tailwind CSS post processing order"

I'm assuming it'll be fixed in a follow up PR?

deboer-tim commented:
> I'm finding it very difficult to read / as though the modal has some form of opacity? I saw the comment: "It is not possible to use bg-opacity-60 with colors from the registry due to Tailwind CSS post processing order"
> 
> I'm assuming it'll be fixed in a follow up PR?

Yes. That's the MessageBox text that's light, because it doesn't have support for light mode yet. This PR is only covering Modal: the dialog outline and background color, and opacity of the rest of the screen. Individual dialogs like MessageBox and others will be covered under #7184.

feloy commented on packages/ui/src/lib/modal/Modal.svelte:
```diff
@@ -35,12 +35,13 @@ if (previously_focused) {
   <button
     aria-label="close"
-    class="fixed top-0 left-0 w-full h-full bg-black bg-opacity-60 bg-blend-multiply z-40"
+    class="fixed top-0 left-0 w-full h-full bg-[var(--pd-modal-fade)] bg-blend-multiply z-40"
+    style="opacity: 0.6"
```
It seems to work by using class `opacity-60` (instead of `bg-opacity-60`)

```suggestion
    class="fixed top-0 left-0 w-full h-full bg-[var(--pd-modal-fade)] bg-blend-multiply z-40 opacity-60"
```

deboer-tim replied:
So focused on replicating what was there I failed to see the obvious. 🤦🏼‍♂️ Fixed.

deboer-tim commented:
Rebased to fix merge conflict from other light mode changes. Also fixed a nit that was annoying me while retesting: the background should not have a button cursor.

@cdrage and @feloy please re-review. Again, this is just the modal background, edge, and rest of the screen, not any particular dialog contents.
